src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -616,6 +658,10 @@ function MoneyRequestConfirmationList({
                 return;
             }
 
+            if (formError) {
```

Comment:
We caused a bug. RCA here - https://github.com/Expensify/App/issues/41336#issuecomment-2098318729